### PR TITLE
R111 merge in AB#53369 card component editor visibility

### PR DESCRIPTION
**Pull request by phudson-he**
Fix for ensuring the card component editor is visible

### CODE REVIEW

**Comment by aj-he** [comment]
@phudson-he @csmith-he I'm happy that all of those commits from @khodgkinson-he as the onenterkey binding was only added back to acc as it was not being implemented immediately in keystone.

File: arches/app/media/js/views/base-manager.js [hunk 1]
```diff
@@ -64,6 +64,20 @@ define([
                 return window.location.pathname === "/search" || window.location.pathname === "/plugins/c8261a41-a409-4e45-b049-c925c28a57da";
             });
 
+            // Register binding of onEnterkeyClick. e.g. <div data-bind="onEnterkeyClick"> </div>
```
Comment (csmith-he) [inline arches/app/media/js/views/base-manager.js:67]: @phudson-he - Is this part of the solution for the visibility issue solution?

Comment (aj-he) [reply arches/app/media/js/views/base-manager.js:67]: @csmith-he see my comment above

Comment (csmith-he) [reply arches/app/media/js/views/base-manager.js:67]: Re: @aj-he's comment, I'll approve this then

Comment (phudson-he) [reply arches/app/media/js/views/base-manager.js:67]: No it isnt

**Review by csmith-he — APPROVED** [review]

**Review by Rehaan-Arif — APPROVED** [review]
Tested successfully

**Review by aj-he — APPROVED** [review]
all good

**Pull request merged**